### ISSUE TITLE
su/bin permission denied

### ISSUE THREAD

**Issue by changtj** [meefik/linuxdeploy#922]
android 7

install type ：directoy

[03:44:40] :: Configuring extra/dbus ... 
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] ## desktop/lxde : do_install && do_configure
[03:44:40] :: Installing desktop/lxde ... 
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] :: Configuring desktop/lxde ... 
[03:44:40] ## desktop : do_install && do_configure
[03:44:40] ## extra/ssh : do_install && do_configure
[03:44:40] :: Installing extra/ssh ... 
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] ## graphics/vnc : do_install && do_configure
[03:44:40] :: Installing graphics/vnc ... 
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] :: Configuring graphics/vnc ... 
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] ln: /data/local/mnt/.vnc/xstartup: Operation not permitted
[03:44:40] chroot: can't execute '/bin/su': Permission denied
[03:44:40] ## graphics : do_install && do_configure
[03:44:40] <<< deploy

**Comment by changtj** [comment]
again
install distribution：ubuntu
install type ：directory
install path：/sdcard/xxx/Linux Deploy/ubuntu
settings：
env：
/sdcard/xxx/linuxdeploy/env

install process
...
[04:08:55] I: Extracting util-linux...
[04:08:56] I: Extracting zlib1g...
[04:08:56] ## core/emulator : do_install && do_configure
[04:08:56] ## core/mnt : do_install && do_configure
[04:08:56] :: Configuring core/mnt ... 
[04:08:56] ## core/net : do_install && do_configure
[04:08:56] :: Configuring core/net ... 
[04:08:56] chroot: can't execute '/debootstrap/debootstrap': Permission denied
[04:08:56] <<< deploy

**Comment by SakiiCode** [comment]
Is your phone rooted correctly?

**Comment by changtj** [comment]
@SakiiCode yes,with supersu,and installed sucessly with  install image type.

**Comment by meefik** [comment]
Can not be installed in the directory on sdcard with FAT. Please use internal path (/data/local) or other installation type.
